### ISSUE TITLE
iBeacon Bug with iOS10 - Fail Updating SG and Temp

### ISSUE THREAD

**Issue by noahbaron** [tbryant/brewometer-cordova#2]
Current Workaround:

Update 2: If I uninstalled all iBeacon-detecting apps, rebooted, and then installed only one such app, it seems to work fine.

Suspect issue is with Apple, not with Cordova plugin

**Comment by tbryant** [comment]
evidence of other people having iOS 10 issues...
possibly relevant thread:
https://forums.developer.apple.com/thread/63792

http://stackoverflow.com/questions/39638067/swift2-3-code-for-beacon-detection?noredirect=1#comment66581038_39638067

**Comment by noahbaron** [comment]
yeah, it's odd that there are issues considering no real changes were made
to the iBeacon functionality

**Comment by tbryant** [comment]
I found this, which may be related
http://stackoverflow.com/questions/38410159/cordova-app-hanging-during-startup-on-ios-10-beta
